org.eclipse.xtext.xbase/src/org/eclipse/xtext/xbase/imports/ImportOrganizer.java

```diff
@@ -56,7 +57,16 @@
 		if (unresolvedTypeResolver != null)
 			unresolvedTypeResolver.resolve(typeUsages, resource);
 		Map<String, JvmDeclaredType> name2type = conflictResolver.resolveConflicts(typeUsages, nonOverridableTypesProvider, resource);
-		return getOrganizedImportChanges(resource, name2type, typeUsages);
+		List<ReplaceRegion> changes = getOrganizedImportChanges(resource, name2type, typeUsages);
```

Comment:
I think it's ok the way it is. A method 'removeNullEdits' would be a plus, though.